Sources/ProjectDescription/AnalyzeAction.swift

```diff
@@ -7,10 +7,6 @@ public struct AnalyzeAction: Equatable, Codable {
     /// Indicates the build configuration the product should be analyzed with.
     public let configuration: ConfigurationName
 
-    init(configuration: ConfigurationName) {
```

Comment:
SwiftLint fixes are odd. I need to investigate them.

Reply:
I think default init are now authomatically synthesized

Reply:
Oh I see! That might be the issue then. I need to double check if the synthesized initializers will cause breaking changes.

Reply:
As long as they don't include any custom defaults, they shouldn't. From what I can see, they should be fine imho